packages/react/src/components/AnimatedSprite.js

```diff
@@ -24,10 +24,21 @@ const AnimatedSprite = (root, props) =>
     animatedSprite[isPlaying ? 'gotoAndPlay' : 'gotoAndStop'](initialFrame || 0);
     animatedSprite.applyProps = (instance, oldProps, newProps) =>
     {
-        const { textures, isPlaying, initialFrame, ...props } = newProps;
+        const { textures, isPlaying, initialFrame, images, ...props } = newProps;
 
         let changed = applyDefaultProps(instance, oldProps, props);
 
+        if (images && oldProps.images !== images)
```

Comment:
Seems like `oldProps.images !== images` should be enough.

Reply:
Hmm - I can’t remember why I added this.

Reply:
Because `images` _needs_ to exist for the logic in the conditional to function. If it doesn't exist, skip the function.

Reply:
Ah I see - that’s right’. So it’s fine as is.  Can we merge?

Reply:
I approved it a while back. I'm not sure about the release process though, I just assist on maintenance and such. Do we know who is in charge of merges and releases?

Reply:
Haven’t the slightest idea.